### PR TITLE
refactoring riak, examples, timeouts

### PR DESCRIPTION
**Pull request by cipy**
bumping version to 1.2.1

### CODE REVIEW

File: database/riak.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 description:
      - This module can be used to join nodes to a cluster, check
        the status of the cluster.
-version_added: "1.2"
+version_added: "1.2.1"
```
Comment (bcoca) [inline database/riak.py:28]: this should be 2.0 now

File: database/riak.py [hunk 2]
```diff
@@ -56,14 +56,14 @@
...
-    default: null
+    default: "yes", infinity
     aliases: []
     type: 'int'
   wait_for_ring:
     description:
       - Number of seconds to wait for all nodes to agree on the ring.
     required: false
-    default: null
+    default: "yes", infinity
```
Comment (bcoca) [inline database/riak.py:66]: a better documentation might be:
default: null (infinity)

File: database/riak.py [hunk 3]
```diff
@@ -161,10 +161,10 @@ def main():
 
     node_name = stats['nodename']
     nodes = stats['ring_members']
-    ring_size = stats['ring_creation_size']
+    ring_size = stats['ring_num_partitions']
```
Comment (bcoca) [inline database/riak.py:164]: I'm not sure I understand this change

**Comment by bcoca** [comment]
this needs a rebase (file was moved to databases/misc/riak.py

**Comment by gregdek** [comment]
@cipy A friendly reminder: this pull request has been marked as needing your action, and it hasn't been updated in the last month. 

If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we'll keep it open pending your changes.

If we receive no indication from you within a week, we will close this pull request.

**Comment by gregdek** [comment]
Closing this PR due to inactivity. Sorry @cipy -- if you believe we've closed this in error, please feel free to reopen/rebase or create a new PR.

**Pull request closed without merging**